frontend/src/host_orchestrator/orchestrator/instancemanager.go

```diff
@@ -140,6 +143,28 @@ func NewCVDToolInstanceManager(opts *CVDToolInstanceManagerOpts) *CVDToolInstanc
 	}
 }
 
+func newCVDExecContext(execContext ExecContext, user string) CVDExecContext {
+	if user != "" {
+		// Without a user, cvd is executed as the current user.
```

Comment:
let's add this comment as a general comment for the whole `newCVDExecContext` function.

Reply:
Done